website_attribute_set/models/mixins.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2011 Kencove (http://www.kencove.com).
+# @author Mohamed Alkobrosli <malkobrosly@kencove.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from difflib import SequenceMatcher
+
+from odoo import api, models
+from odoo.osv import expression
+
+_logger = logging.getLogger(__name__)
+
+
+def search_extra(env, search_term):
+    extra_domains = []
+    attributes = (
+        env["attribute.attribute"].sudo().search([("e_com_visibility", "=", True)])
+    )
+    for attribute in attributes:
+        if attribute.attribute_type in ["char", "text"]:
+            extra_domain = [(attribute.name, "ilike", search_term)]
+            extra_domains.append(extra_domain)
+        elif attribute.attribute_type in ["integer", "float"]:
+            try:
+                if attribute.attribute_type == "integer":
+                    extra_domain = [(attribute.name, "ilike", int(search_term))]
+                    extra_domains.append(extra_domain)
+                elif attribute.attribute_type == "float":
+                    extra_domain = [(attribute.name, "ilike", float(search_term))]
+                    extra_domains.append(extra_domain)
+            except ValueError as e:
+                _logger.info(f"{e}")
```

Comment:
```suggestion
                _logger.warning(f"{e}")
```
This is an error value that should be warning instead of logged as information.

Reply:
remember it iw a website, and website users must do mistakes but we must ignore their mistakes as we expect and handle properly it should be our job in the background

Reply:
I am even thinking of not logging the error in production as it can come from the seach function in website which should be not important to log